internal/api/hcpopenshiftcluster_test.go

```diff
@@ -35,6 +36,7 @@ func newTestValidator() *validator.Validate {
 		arm.ManagedServiceIdentityTypeSystemAssigned,
 		arm.ManagedServiceIdentityTypeSystemAssignedUserAssigned,
 		arm.ManagedServiceIdentityTypeUserAssigned))
+	validate.RegisterAlias("enum_optionalclustercapability", EnumValidateTag(generated.PossibleOptionalClusterCapabilityValues()...))
```

Comment:
nit: There's nothing technically wrong with this, but for consistency's sake I'd prefer the unversioned enum here (`OptionalClusterCapabilityImageRegistry`).